core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/LibraryDesign.java

```diff
@@ -31,13 +29,11 @@ public static boolean validate(Library library, Iterable<LibraryDesign> rules) {
   private Long librarySelectionType;
   @Column(nullable = false)
   private Long libraryStrategyType;
+  @OneToOne(targetEntity = LibraryType.class)
+  @JoinColumn(name = "libraryType", nullable = false)
+  private LibraryType libraryType;
   @Column(nullable = false)
   private String name;
-  @Column(nullable = false)
-  private Boolean paired;
-  @Enumerated(EnumType.STRING)
-  @Column(nullable = false)
-  private PlatformType platformName;
   @OneToOne(targetEntity = SampleClassImpl.class)
```

Comment:
Yes. The library type will be either Single, Mate Paired, or Paired End. I'm still unclear on how mate paired is different from paired end. There isn't an explicit `paired` column in the `LibraryType`s, but I can add one.